mail-mta/ssmtp/ssmtp-2.64_p11-r1.ebuild

```diff
@@ -47,6 +44,10 @@ RDEPEND="
 	)
 "
 
+PATCHES=(
```

Comment:
I dare say it's a little bit confusing if PATCHES doesn't mention Debian patches being applied. Maybe at least add a comment regarding that.

Reply:
I've mentioned it in the eclass. Do you want me to mention it in ssmtp too? I'm not sure there's a need given the eclass name includes 'patches' now too.

Reply:
Not saying it's necessary but it could make things clearer.

Reply:
Not sure about that but you could even go further and just put the patch-directory path in a variable, and let user use it explicitly. Then it'd allow deciding whether Debian patches should go before or after ebuild patches.